lib/releaselib.sh

```diff
@@ -450,6 +450,15 @@ release::gcs::copy_release_artifacts() {
   local gce_path=$release_stage/full/kubernetes/cluster/gce
   local gci_path
 
+  logecho "Publish release artifacts to gs://$bucket..."
+
+  logecho "- Checking whether $gcs_destination already exists..."
+  if $GSUTIL ls $gcs_destination >/dev/null 2>&1 ; then
```

Comment:
This is (sadly?) consistent with the remainder of the file, or I'd agree and make it `"${gcs_destination}"`.